src/workerd/io/compatibility-date.capnp

```diff
@@ -245,4 +245,13 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # Enables TCP sockets in workerd.
   # These are still under development and therefore subject to change.
   # WARNING: DO NOT depend on this feature as its API is still subject to change.
+
+  specCompliantResponseRedirect @23 :Bool
+      $compatEnableDate("2023-03-14")
```

Comment:
If this feateris ready for internal use / testing, feel free to add the `$experimental;` annotation.